test/models/term_test.exs

```diff
@@ -36,9 +37,39 @@ defmodule CoursePlanner.TermTest do
       %{
         name: "Spring",
         start_date: %{day: 17, month: 4, year: 2010},
-        end_date: %{day: 17, month: 3, year: 2010}
+        end_date: %{day: 17, month: 3, year: 2010},
+        minimum_teaching_days: 5,
       }
     changeset = Term.changeset(%Term{}, invalid_attrs)
     refute changeset.valid?
   end
+
+  test "term has minimum teaching days" do
+    {:ok, term} = Terms.create(%{
+      "name" => "Spring",
+      "start_date" => %{day: 17, month: 3, year: 2017},
+      "end_date" => %{day: 22, month: 3, year: 2017},
+      "minimum_teaching_days" => 3,
+      "holidays" => %{
+        "0" => %{name: "Labor Day 1", date: %{day: 18, month: 3, year: 2017}},
+        "1" => %{name: "Labor Day 2", date: %{day: 19, month: 3, year: 2017}}
+      }
+    })
+    assert length(term.holidays) == 2
+  end
+
+  test "term doesn't have minimum teaching days" do
```

Comment:
could you add a test which has no holidays and the term doesn't have enough teaching days anyways?